ui/app/models/auth-method.js

```diff
@@ -110,6 +104,10 @@ const ModelExport = AuthMethodModel.extend({
   }),
   canDisable: alias('deletePath.canDelete'),
   canEdit: alias('configPath.canUpdate'),
+
+  tune(data) {
+    return this.store.adapterFor('auth-method').tune(this.path, data);
+  },
```

Comment:
This replaces the `memberAction` method which was a part of `ember-api-actions`. Essentially it just makes a request to the backend and since it was only used in this one instance I removed the package and added a method to the adapter to achieve the same thing.